cmd/copyEnumeratorInit.go

```diff
@@ -52,7 +52,7 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 
 	// Ensure we're only copying from a directory with a trailing wildcard or recursive.
 	isSourceDir := traverser.isDirectory(true)
-	if isSourceDir && !cca.recursive && !cca.copyContents {
+	if isSourceDir && !cca.recursive && !cca.stripTopDir {
```

Comment:
Please see new question in my earlier thread about this (from first-pass review)